src/install.html

```diff
@@ -88,29 +86,24 @@
             />
           </div>
           <div class="col-md-8 order-md-1 col-lg-7 text-center text-md-start">
-            <h1 class="mb-3">Install Artichoke Ruby</h1>
-            <p class="lead mb-4">
-              Artichoke Ruby installs on Linux, macOS, and Windows. New builds
-              are pushed nightly.
-            </p>
-            <p class="lead mb-4">
-              Try out a build using one of these installation methods.
-            </p>
+            <h1 class="mb-3"><%= it.t.install.title %></h1>
+            <p class="lead mb-4"><%= it.t.install.install %></p>
```

Comment:
I pulled down this branch and tried it locally. It looks like the `h1` should be `it.t.install.install` and the `p` should be `it.t.install.targets`. I'll push up a fix.

Reply:
<img width="1193" alt="Screen Shot 2022-01-22 at 10 02 23 AM" src="https://user-images.githubusercontent.com/860434/150650473-91c943f7-aa59-4524-b7c0-bd97a4eac479.png">

this was the difference when comparing production and this branch side by side.